src/common/components/vanilla/testimonial/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react';
```

Comment:
good point.

the intention is that it will become a vanilla component... but it isn't yet.

I can move it.